src/transformStateWithClones.js

```diff
@@ -6,8 +6,32 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  let newState = { ...state };
+  const result = [];
```

Comment:
`res/result/arr/obj/el/elem/etc` are bad names, the variable name should be informative, for example `statesHistory`